client/components/product-card/README.md

```diff
@@ -94,16 +94,17 @@ export default class extends React.Component {
 				}
 			>
 				<ProductCardOptions
-					billingTimeFrame={ 'per year' }
 					optionsLabel="Backup options:"
 					options={ [
 						{
+							billingTimeFrame={ 'per year' }
 							discountedPrice: 12,
 							fullPrice: 14,
 							slug: 'jetpack_backup_daily_monthly',
 							title: 'Daily Backups',
 						},
 						{
+							billingTimeFrame={ 'per year' }
```

Comment:
should this be `per month`?

Reply:
Well, probably yes 😀For the sake of clarity, I'll update the readme so that this is consistent across the example code.